### DISCUSSION TITLE
Consulta QMP5

### DISCUSSION THREAD

**Discussion by FacundoPugliese** [dds-jv/foro#240] [Consultas de objetos] [answered]
Saludos!

En el enunciado de la tarea semanal se pide como requerimiento las propuestas de quitar o agregar prendas de un determinado guardarropas por parte de otro usuario. Las mismas hacen referencia a los guardarropas compartidos entre ambos usuarios cierto (requerimiento 2)? 
En otras palabras yo, como usuario, solo puedo sugerirle a otro agregar o quitar una prenda sobre un guardarropa compartido entre ambos no?

**Comment by roli93** [comment, marked as answer]
Exactamente
